### ISSUE TITLE
Modules not found

### ISSUE THREAD

**Issue by yieniggu** [celo-org/rainbowkit-celo#26]
After installing package using yarn:

`yarn add @celo/rainbowkit-celo @rainbow-me/rainbowkit wagmi ethers
`
And using code provided from this (demo)[https://rainbowkit-with-celo.vercel.app/]:
```
import { RainbowKitProvider, ConnectButton } from "@rainbow-me/rainbowkit";
import { configureChains, createClient, WagmiConfig } from "wagmi";
import { jsonRpcProvider } from "wagmi/providers/jsonRpc";
import celoGroups from "@celo/rainbowkit-celo/lists";
import { Alfajores, Celo } from "@celo/rainbowkit-celo/chains";

// (NEW) rainbow>=0.8.1 && wagmi >= 0.9.0
const { chains, provider } = configureChains(
  [Alfajores, Celo],
  [
    jsonRpcProvider({
      rpc: (chain) => ({ http: chain.rpcUrls.default.http[0] }),
    }),
  ]
);

const connectors = celoGroups({ chains });

const wagmiClient = createClient({
  autoConnect: true,
  connectors,
  provider,
});

export default function Wrap() {
  return (
    <WagmiConfig client={wagmiClient}>
      <RainbowKitProvider chains={chains}>
        <YourApp />
      </RainbowKitProvider>
    </WagmiConfig>
  );
}

function YourApp() {
  return <ConnectButton />;
}
```

Throws this errors:

```
Compiled with problems:X

ERROR in ./src/App.js 15:21-30

export 'Alfajores' (imported as 'Alfajores') was not found in '@celo/rainbowkit-celo/chains' (module has no exports)


ERROR in ./src/App.js 15:32-36

export 'Celo' (imported as 'Celo') was not found in '@celo/rainbowkit-celo/chains' (module has no exports)


ERROR in ./src/index.js 12:33-36

export 'App' (imported as 'App') was not found in './App' (possible exports: default)


ERROR in ./node_modules/@celo/rainbowkit-celo/chains/index.js 1:0-51

Module not found: Error: Can't resolve './alfajores' in '/home/basti/Desktop/kolor/rainbow-celo/node_modules/@celo/rainbowkit-celo/chains'
Did you mean 'alfajores.js'?
BREAKING CHANGE: The request './alfajores' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.


ERROR in ./node_modules/@celo/rainbowkit-celo/chains/index.js 2:0-41

Module not found: Error: Can't resolve './celo' in '/home/basti/Desktop/kolor/rainbow-celo/node_modules/@celo/rainbowkit-celo/chains'
Did you mean 'celo.js'?
BREAKING CHANGE: The request './celo' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.


ERROR in ./node_modules/@celo/rainbowkit-celo/chains/index.js 3:0-47

Module not found: Error: Can't resolve './baklava' in '/home/basti/Desktop/kolor/rainbow-celo/node_modules/@celo/rainbowkit-celo/chains'
Did you mean 'baklava.js'?
BREAKING CHANGE: The request './baklava' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.


ERROR in ./node_modules/@celo/rainbowkit-celo/lists/index.js 3:0-67

Module not found: Error: Can't resolve '@celo/rainbowkit-celo/wallets' in '/home/basti/Desktop/kolor/rainbow-celo/node_modules/@celo/rainbowkit-celo/lists'
Did you mean 'index.js'?
BREAKING CHANGE: The request '@celo/rainbowkit-celo/wallets' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

```

**Comment by aaronmgdr** [comment]
Thanks for bringing this up. It looks like this is due to it being an ESM module. 

To make sure I get this fixed....Whats the bundler you are using? Webpack 5? create-react-app?

I will work on getting out a preliminary fix in the next few days. 

For the time being you might be best off just copying the config files you need.

**Comment by yieniggu** [comment]
Yes, webpack5 & create-react-app.

I tried copying files, unfortunately other errors are raising now.

**Comment by aaronmgdr** [comment]
Ive released 0.8.1-beta.3  installable with. `yarn add` or `npm install` `@celo/rainbowkit-celo@beta` 

I install on a fresh create-react-app and it all seems to work. 

will release prod version shortly
